.travis.yml

```diff
@@ -5,7 +5,7 @@ python:
   - "3.4"
   - "3.5"
 env:
-  - NUMPY="numpy==1.7"
+  - NUMPY="numpy==1.9"
```

Comment:
numpy is unpinned in one part of the test matrix

we test against 1 pinned version and latest, so currently this is 1.9 and 1.11